PAS-65: Add Gradebook base page

BraidenCutforth opened the pull request:
- Add gradebook base page
- Consider some basic screen sizing
- Build template areas for page base using css grid

daelynj commented:
@BraidenCutforth let's hold off on merging this; there is something going on with jest and typescript. I'll get to the bottom of it before we add any dependencies to master.

BraidenCutforth committed:
PAS-64: Add Gradebook base page

- Add gradebook base page
- Consider some basic screen sizing
- Build template areas for page base using css grid

dereklowlind reviewed (approved):
That is a colourful page! Totally works though and is ready to merge

daelynj commented on src/GradeBook.tsx:
```diff
@@ -0,0 +1,77 @@
+	grid-area: header;
+`;
+
+function GradeBook() {
```
Any particular reason why we're not using arrow functions and forgoing the return statement?

```suggestion
const GradeBook = () => (
    <GradeBookContainer>
        <LogoContainer>
            <Logo width="100%" height="100%" />
        </LogoContainer>
	<Header />
	<Sidebar />
	<MainContent />
        <Account />
    </GradeBookContainer>
);
```

BraidenCutforth replied:
I don't like to use arrow functions when it doesn't pertain to the context of `this` usually. It is honestly a preference over functionality. https://stackoverflow.com/questions/49306148/why-is-arrow-syntax-preferred-over-function-declaration-for-functional-react-com

I don't necessarily agree with the arguments made either way, and I think there really isn't a strong argument for either.

daelynj reviewed (approved):
👍
